src/logger/index.ts

```diff
@@ -20,28 +27,35 @@ const logLevelMapping = {
 
 const LOG_LEVEL = config.logger.level && logLevelMapping[config.logger.level] || LogLevel.important;
 const LOG_TO_FILES = config.logger.logToFiles || false;
+// LOG_PATH defaults to `ceramic-anchor-service/logs/`
 let LOG_PATH = config.logger.filePath || path.join(path.resolve(__dirname, '../../'), 'logs/');
 if (!LOG_PATH.endsWith('/')) {
   LOG_PATH = LOG_PATH + '/';
 }
 
-const ACCESS_FILE_PATH = LOG_PATH + 'access.log';
-const METRICS_FILE_PATH = LOG_PATH + 'metrics.log';
-const EVENTS_FILE_PATH = LOG_PATH + 'events.log';
+const ACCESS_FILE_PATH = path.join(LOG_PATH, 'access.log');
+const EVENTS_FILE_PATH = path.join(LOG_PATH, 'events.log');
+const METRICS_FILE_PATH = path.join(LOG_PATH, 'metrics.log');
+const STDOUT_FILE_PATH = path.join(LOG_PATH, 'stdout.log');
+
 const REMOVE_TIMESTAMP = true;
 
-const LOG_FMT = '[:date[web]] ip=:remote-addr ts=:date[iso] method=:method path=:url http_version=:http-version req_header:req[header] status=:status content_length=:res[content-length] content_type=":res[content-type]" ref=:referrer user_agent=:user-agent elapsed_ms=:total-time[3]';
+const ACCESS_LOG_FMT = 'ip=:remote-addr ts=:date[iso] method=:method path=:url http_version=:http-version req_header:req[header] status=:status content_length=:res[content-length] content_type=":res[content-type]" ref=:referrer user_agent=:user-agent elapsed_ms=:total-time[3]';
 
 /**
  * Logs to the console based on log level
  */
 class ConsoleLogger {
```

Comment:
Now that this does more than just log to the console, should we call it 'DiagnosticsLogger'?